app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for current_user.active_relationships.build, remote: true do |f| %>
```

Comment:
giải thích cơ chế về rails ajax và remote_true cho t nhé

Reply:
chưa rõ ràng lắm, a nói kĩ hơn về cái rails a ajax được không ạ

Reply:
- Sử dụng remote: true, thì nó sẽ gọi tới action trong from_for, sau đó xử lý và gọi tới file js.erb với tên tương ứng với tên action, nó thực hiện lệnh trong đó mà cập nhập lại trang hiện tại không cần reload.

Reply:
Theo a tìm hiểu, thì do mình sử dụng turbolink, khi bạn click vào thẻ bất kì trên trang thay vì redirect tới một trang khác thì Turbolinks sẽ tạo một Ajax request, khi nhận xết quả trả về nó sẽ thay thế toàn bộ kết quả trong thẻ body. Nên xảy ra lỗi ở hành động thứ 2 của turbolink. : a hiểu là có 2 lần gửi request 1 lần của ajax cho remote true, 1 lần của turbolink.

Reply:
không phải đâu nhé, a xem a có đang require cả jquery_ujs và rails-ujs không a?
nếu đúng thì cả 2 cái dùng gửi request cho remote true nên ính ra 2 request.
chỗ này a đưa `current_user.active_relationships.build` vào controller nhé